ctpolicy/ctpolicy.go

```diff
@@ -205,6 +206,15 @@ func lifetimeInMonths(cert *x509.Certificate) int {
 	return lifetimeInMonths
 }
 
+// lifetimeInDays calculates and returns cert lifetime expressed in days
+// flooring incomplete days.
+func lifetimeInDays(cert *x509.Certificate) int {
+	start := cert.NotBefore
+	end := cert.NotAfter
+	days := end.Sub(start).Hours() / 24
```

Comment:
Ah right. I have changed how the policy determines how many submissions are required. It will compare the lifetime in seconds to 180 days in seconds instead. This will also prevent any rounding errors from occurring.

Reply:
That should be equivalent, I believe?

https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/time/time.go;l=791-795

Reply:
@pphaneuf You are right. The `Sub()` should have handled the daylight savings time zone transitions.
https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/time/time.go;l=880-907

@breadyzhang Calculations in both hours and seconds should be correct. See if you want to revert to the previous commit.